src/Stratis.Bitcoin.Features.PoA/FederationManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Configuration;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class FederationManager
```

Comment:
Why do you need a manager I wonder.

Reply:
To check if we are federation and provide this info to other components that may or may not work based on that information (or to initialize some components for being federation member and not initialize if normal node. miner class- for example)